### PR TITLE
doc: Contributor Over Time

### PR DESCRIPTION
**Pull request by Miss-you**
### What this PR does / why we need it:

add picture about Contributor Over Time

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible?

### CODE REVIEW

**Comment by membphis** [comment]
image

**Comment by Miss-you** [comment]
I have fix the problem in the image attributor-over-time.png.

image

**Review by membphis — APPROVED** [review]

**Comment by moonming** [comment]
How to update this?

**Comment by membphis** [comment]
It can only be updated manually now

**Comment by Miss-you** [comment]
> How to update this?

You expect to be able to automatically update contributor trend charts on a daily or monthly, right?

**Comment by Miss-you** [comment]
> It can only be updated manually now



> It can only be updated manually now

Yes, but if needed, I could add a regular update feature, something like startchart.

**Comment by moonming** [comment]
> > How to update this?
> 
> You expect to be able to automatically update contributor trend charts on a daily or monthly, right?

yes, monthly will be enough

**Review by moonming — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -288,6 +288,10 @@ A wide variety of companies and organizations use APISIX for research, productio
 
 Users are encouraged to add themselves to the [Powered By](doc/powered-by.md) page.
 
+## Contributor Over Time
+
+![](./doc/images/contributor-over-time.png)
```
Comment (juzhiyuan) [inline README.md:293]: We’d better add some descriptions here, so when this image is broken due to network, this description will show properly.

Comment (membphis) [reply README.md:293]: nice catch

Comment (moonming) [reply README.md:293]: @Miss-you ping

**Pull request merged**